tests/lean/infoTree.lean

```diff
@@ -2,20 +2,13 @@ import Lean
 
 open Lean.Elab
 
-elab "enableInfo!" : command  => enableInfoTree
-
-elab "showInfoTrees!" : command => do
-  let trees ← getInfoTrees
-  trees.forM fun tree => do
-    logInfo f!"{← tree.format}"
-
 structure A where
   val : Nat → Nat
 
 structure B where
   pair : A × A
 
-enableInfo!
+set_option trace.Elab.info true
```

Comment:
@leodemoura I had to change this because with command-level info trees, the info state is reset before `showInfoTrees!` is run, so its output is always empty. This solution should be more general in any case.

Reply:
That is fine. `elabInfo!` was just a hack to test the `infoTree` before it was connected with the IDEs